content/docs/Configuration/examples.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Examples 
+date: 2022-09-22
+weight: 1
+---
+
+# Configuration examples 
+
+## Examples for actions
+You can find detailed documentation for actions [here](/docs/actions).
+
+### Getting version
+{{< details title="Getting version from specfile" >}}
+```yaml
+    get-current-version:
+    - grep -oP '^Version:\s+\K\S+' my-package.spec
```

Comment:
```
rpmspec -q --queryformat "%{VERSION}\n" *spec |head -n1
```
will honor possible macros. This command is part of `rpm-build` package.

Reply:
thanks! added